src/class_singleWorker.py

```diff
@@ -1266,7 +1267,7 @@ def sendMsg(self):
                     except:
                         apiNotifyPath = ''
                     if apiNotifyPath != '':
-                        call([apiNotifyPath, "newMessage"])
+                        call([apiNotifyPath, "newMessage"])  # nosec
```

Comment:
can you remove nosec here please?

Reply:
So it's not a magic comment turning off some codacy check like `# noqa`?

Reply:
@g1itch: It is exactly that. Peter wants to wait to review the security implications of this class of warning before silencing them.